src/main/webapp/stylesheets/teammatesCommon.css

```diff
@@ -1184,14 +1184,33 @@ Prevent Long URL From Breaking Out of Container
     margin-right: 5px;
 }
 
-/*
-Glyphicon for success status messages
-*/
+/*//////////////////////////////////////////////////////////
+////////////////////// Status Icons //////////////////////
+//////////////////////////////////////////////////////////*/
 
-.icon-success:before {
+.icon-success:before,
+.icon-warning:before,
+.icon-danger:before,
+.icon-info:before {
+    /*noinspection CssNoGenericFontName*/
```

Comment:
I don't disagree with the fact that if IDEs can be made optional it can be nice though.

Though saying that an IDE's usage should never be a requirement is too far of a stretch (some projects may simply want consistency throughout and depend on the IDE tools to achieve some things, and they can't be achieved outside of the IDE, or at least easily).